apiary.apib

```diff
@@ -1336,7 +1336,11 @@ Obtains information about a bucket.
                             "provider": "user",
                             "timestamp": "2019-08-12T14:36:08+0200"
                         }
-                ]
+                ],
+                "owner": {
```

Comment:
huh, on ten owner se asi ukaze ve vice operacich (`toApiResponse()`) je tam proste pouzita vsude mozne. Ale koukam na to a je v tom strasnej bordel, takze bych se tim asi moc netrapil

Reply:
Ano máš pravdu a ano, koukal jsem na to úplně stejně :-)

Reply:
Ještě sem to přidal do bucket listu s dvěma ukázkama.

Reply:
pak su happy jak dva grepy